docroot/themes/contrib/civictheme/civictheme_library/components/03-organisms/mobile-navigation/mobile-navigation.stories.js

```diff
@@ -13,8 +13,8 @@ export default {
 
 export const MobileNavigation = () => {
   const generalKnobTab = 'Menu';
-  const topMenuKnobTab = 'Top menu';
-  const bottomMenuKnobTab = 'Bottom menu';
+  const topMenuKnobTab = 'Top block';
```

Comment:
please revert these as in Library they are menus (there is no idea of a block in library)